modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -912,3 +912,12 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 /datum/loadout_item/under/formal/basil
 	name = "Basil Uniform"
 	item_path = /obj/item/clothing/under/costume/basil_boys
+
+/datum/loadout_item/under/formal/demon
+	name = "Demonic Suit"
+	item_path = /obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon
+
+/datum/loadout_item/under/formal/designer_demon
+	name = "Designer Demonic Suit"
```

Comment:
```suggestion
	name = "Designer Inferno Suit"
```

Reply:
because we gotta follow "POLICY" or whatever


I personally still think all this is fine but
well shit